assets/stylesheets/gu-sass/layout.scss

```diff
@@ -0,0 +1,55 @@
+// This file contains variable and classes definition related to the layout of
+// the project.
+
+// Spacing variables
+// h stands for horizontal
+// v stands for vertical
+// =============================================================================
+$gu-h-spacing: 1.25rem;
```

Comment:
After an offline discussion, we will add `post-css` as a step in our build process for dev and prod environment.